interceptors/logging/logging.go

```diff
@@ -38,6 +38,22 @@ func newCommonFields(kind string, c interceptors.CallMeta) Fields {
 	}
 }
 
+// disableCommonLoggingFields returns copy of newCommonFields with disabled fields removed from the following
+// default list.
+//
+//	-SystemTag[0],
+//	-ComponentFieldKey,
+//	-ServiceFieldKey,
+//	-MethodFieldKey,
+//	-MethodTypeFieldKey,
```

Comment:
Go comment lists have [a standard way](https://tip.golang.org/doc/comment) of being formatted. I think this should be
```suggestion
// disableCommonLoggingFields returns copy of newCommonFields with disabled fields removed from the following
// default list. The following are the default logging fields:
//   - SystemTag[0]
//   - ComponentFieldKey
//   - ServiceFieldKey
//   - MethodFieldKey
//   - MethodTypeFieldKey
```